slides.html

```diff
@@ -41,11 +47,16 @@
 ---
 class: center, middle
 
+$\sum^{n}_{i} \frac{f(x)}{g(x)^2}$
```

Comment:
left-over of a latex experiment? 
(will remove)

Reply:
removing is ok!